### PR TITLE
[Preview] Upgrade to ocamlformat.0.19.0

### PR DESCRIPTION
**Pull request by gpetiot**
This is a preview of the not-yet-released `ocamlformat.0.19.0`, please wait until the package is published in opam to merge this PR. The output is still likely to slightly change before the package is released.

### CODE REVIEW

File: infer/src/IR/CallFlags.ml [hunk 1]
```diff
@@ -22,8 +22,8 @@ type t =
 
 let pp f
     ({cf_assign_last_arg; cf_injected_destructor; cf_interface; cf_is_objc_block; cf_virtual}[@warning
-                                                                                               "+9"])
-    =
+                                                                                               "+9"]
+      ) =
```
Comment (smorimoto) [inline infer/src/IR/CallFlags.ml:26]: This looks a bit weird formatting.

Comment (gpetiot) [reply infer/src/IR/CallFlags.ml:26]: Indeed I will have a look

Comment (jvillard) [reply infer/src/IR/CallFlags.ml:26]: To be fair the formatting was broken before too so this isn't a regression.

Comment (smorimoto) [reply infer/src/IR/CallFlags.ml]: Yes! Definitely! But @gpetiot addressed with it incredibly quickly and nicely!

File: sledge/src/fol/context.ml [hunk 2]
```diff
@@ -9,7 +9,8 @@
 
 open Exp
 
-(* Solution Substitutions =================================================*)
+(* Solution Substitutions
+   =================================================*)
```
Comment (smorimoto) [inline sledge/src/fol/context.ml:13]: As a reminder for the Infer team, these comments may need to be re-styled in the future.

**Comment by jvillard** [comment]
Thanks a lot for this PR, we've upgraded to a slightly different version since (https://github.com/ocaml-ppx/ocamlformat/commit/2596a5c780ab1dfb996a499b1355cdd23d36ce34) that's very close to 0.19.0 so I'm closing this PR.

**Pull request closed without merging**